providers/fips/self_test_data.inc

```diff
@@ -1748,3 +1765,851 @@ static const ST_KAT_ASYM_CIPHER st_kat_asym_cipher_tests[] = {
+    0x85, 0x6b, 0xc1, 0xe0, 0x34, 0x73, 0x3b, 0x04,
+    0x85, 0xda, 0xb1, 0xd0, 0x20, 0xc6, 0xbf, 0x76,
+    0x5f, 0xf6, 0x0b, 0x3b, 0x80, 0x19, 0x84, 0xa9,
+    0x0c, 0x2f, 0xe9, 0x70, 0xbf, 0x1d, 0xe9, 0x70,
+    0x04, 0xa6, 0xcf, 0x44, 0xb4, 0x98, 0x4a, 0xb5,
+    0x82, 0x58, 0xb4, 0xaf, 0x71, 0x22, 0x1c, 0xd1,
+    0x75, 0x30, 0xa7, 0x00, 0xc3, 0x29, 0x59, 0xc9,
+    0x43, 0x63, 0x44, 0xb5, 0x31, 0x6f, 0x09, 0xcc,
+    0xca, 0x70, 0x29, 0xa2, 0x30, 0xd6, 0x39, 0xdc,
+    0xb0, 0x22, 0xd8, 0xba, 0x79, 0xba, 0x91, 0xcd,
+    0x6a, 0xb1, 0x2a, 0xe1, 0x57, 0x9c, 0x50, 0xc7,
+    0xbb, 0x10, 0xe3, 0x03, 0x01, 0xa6, 0x5c, 0xae,
+    0x31, 0x01, 0xd4, 0x0c, 0x7b, 0xa9, 0x27, 0xbb,
+    0x55, 0x31, 0x48, 0xd1, 0x64, 0x70, 0x24, 0xd4,
+    0xa0, 0x6c, 0x81, 0x66, 0xd0, 0xb0, 0xb8, 0x12,
+    0x69, 0xb7, 0xd5, 0xf4, 0xb3, 0x4f, 0xb0, 0x22,
+    0xf6, 0x91, 0x52, 0xf5, 0x14, 0x00, 0x4a, 0x7c,
+    0x68, 0x53, 0x68, 0x55, 0x23, 0x43, 0xbb, 0x60,
+    0x36, 0x0f, 0xbb, 0x99, 0x45, 0xed, 0xf4, 0x46,
+    0xd3, 0x45, 0xbd, 0xca, 0xa7, 0x45, 0x5c, 0x74,
+    0xba, 0x0a, 0x55, 0x1e, 0x18, 0x46, 0x20, 0xfe,
+    0xf9, 0x76, 0x88, 0x77, 0x3d, 0x50, 0xb6, 0x43,
+    0x3c, 0xa7, 0xa7, 0xac, 0x5c, 0xb6, 0xb7, 0xf6,
+    0x71, 0xa1, 0x53, 0x76, 0xe5, 0xa6, 0x74, 0x7a,
+    0x62, 0x3f, 0xa7, 0xbc, 0x66, 0x30, 0x37, 0x3f,
+    0x5b, 0x1b, 0x51, 0x26, 0x90, 0xa6, 0x61, 0x37,
+    0x78, 0x70, 0xa6, 0x0a, 0x7a, 0x18, 0x96, 0x83,
+    0xf9, 0xb0, 0xcf, 0x04, 0x66, 0xe1, 0xf7, 0x50,
+    0x76, 0x26, 0x31, 0xc4, 0xab, 0x09, 0xf5, 0x05,
+    0xc4, 0x2d, 0xd2, 0x86, 0x33, 0x56, 0x94, 0x72,
+    0x73, 0x54, 0x42, 0x85, 0x1e, 0x32, 0x16, 0x16,
+    0xd4, 0x00, 0x98, 0x10, 0x77, 0x7b, 0x6b, 0xd4,
+    0x6f, 0xa7, 0x22, 0x44, 0x61, 0xa5, 0xcc, 0x27,
+    0x40, 0x5d, 0xfb, 0xac, 0x0d, 0x39, 0xb0, 0x02,
+    0xca, 0xb3, 0x34, 0x33, 0xf2, 0xa8, 0x6e, 0xb8,
+    0xce, 0x91, 0xc1, 0x34, 0xa6, 0x38, 0x6f, 0x86,
+    0x0a, 0x19, 0x94, 0xeb, 0x4b, 0x68, 0x75, 0xa4,
+    0x6d, 0x19, 0x55, 0x81, 0xd1, 0x73, 0x85, 0x4b,
+    0x53, 0xd2, 0x29, 0x3d, 0xf3, 0xe9, 0xa8, 0x22,
+    0x75, 0x6c, 0xd8, 0xf2, 0x12, 0xb3, 0x25, 0xca,
+    0x29, 0xb4, 0xf9, 0xf8, 0xcf, 0xba, 0xdf, 0x2e,
+    0x41, 0x86, 0x9a, 0xbf, 0xba, 0xd1, 0x07, 0x38,
+    0xad, 0x04, 0xcc, 0x75, 0x2b, 0xc2, 0x0c, 0x39,
+    0x47, 0x46, 0x85, 0x0e, 0x0c, 0x48, 0x47, 0xdb
+};
+
+static const unsigned char ml_kem_1024_entropy[] = {
+    0x14, 0x7c, 0x03, 0xf7, 0xa5, 0xbe, 0xbb, 0xa4,
+    0x06, 0xc8, 0xfa, 0xe1, 0x87, 0x4d, 0x7f, 0x13,
+    0xc8, 0x0e, 0xfe, 0x79, 0xa3, 0xa9, 0xa8, 0x74,
+    0xcc, 0x09, 0xfe, 0x76, 0xf6, 0x99, 0x76, 0x15
+};
+
+static const unsigned char ml_kem_1024_secret[] = {
+    0x48, 0x9d, 0xd1, 0xe9, 0xc2, 0xbe, 0x4a, 0xf3,
+    0x48, 0x2b, 0xdb, 0x35, 0xbb, 0x26, 0xce, 0x76,
+    0x0e, 0x6e, 0x41, 0x4d, 0xa6, 0xec, 0xbe, 0x48,
+    0x99, 0x85, 0x74, 0x8a, 0x82, 0x5f, 0x1c, 0xd6
+};
+
+static const unsigned char ml_kem_1024_seed[] = {
+    0x7c, 0x99, 0x35, 0xa0, 0xb0, 0x76, 0x94, 0xaa,
```

Comment:
Was there any reason for picking the largest, slowest one? :)

Reply:
Future proofing.  This is going to be the last one rendered obsolete.